### PR TITLE
Enable Travis-CI

### PR DESCRIPTION
**Pull request by suma**
I setuped [Travis-CI](https://travis-ci.org/) and [Coveralls](https://coveralls.io/) services.
This PR enables CI with collect coverage and adds build badge to README.md.

This PR doesn't contain these check

* golint
* go vet

### CODE REVIEW

File: test_cover.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#/bin/sh
+# http://www.songmu.jp/riji/entry/2015-01-15-goveralls-multi-package.html
+set -e
+# 中断した時とかにゴミが残らないようにcleanupする
```
Comment (disktnk) [inline test_cover.sh]: please write comment in English (e.g. clean-up if interrupt the process)
L15 is same

Comment (suma) [inline test_cover.sh]: I wrote in English.

**Comment by nobu-k** [comment]
Some tests sometimes fail due to concurrency and we need to fix them before we merge this PR.

**Comment by disktnk** [comment]
refs #50

**Pull request merged**